.gitignore

```diff
@@ -13,3 +13,4 @@ db
 *.orig
 .idea
 tests/*/coverage/
+.vscode/
```

Comment:
why still .vscode added?

Reply:
That's the `gitignore` so it's fine, except it's missing a trailing newline

Reply:
i mean .vscode/ is here, but the vscode is on the commit.